NEWS.md

```diff
@@ -1,60 +1,55 @@
-3.23.1
+3.24.1
 ======
 
+## Highlights
 
-## Features
-
- * The `redis()` destination now handles any number of command parameters.
-   (#2816)
-
- * The `format()` option of `date-parser()` supports lists. From now on, a
-   single `date-parser()` instance is able to process different date formats,
-   making it easy to catch on when some programs change the way they log
-   information. (#2779)
+ * Add a new template function called `format-flat-json()`, which generates
+   flattened json output. This is useful for destinations, where the json
```

Comment:
we generally use the foo() format for destinations and sources, but format-flat-json is a template function, so it should be referenced as $(format-flat-json). and I still think this naming is unfortunate.

Reply:
Thanks fixed! :)

Reply:
@bazsi : don't forget that you just skip one of the key question when we discussed why this naming could be better than the other  (this is my understanding)

Reply:
I think I just said that it is an implementation detail how to handle the flattened/non-flattened modes of operation in clean code. The argument against was not usability rather implementation complexity.